docker-compose-tests.yml

```diff
@@ -1,16 +1,19 @@
 version: '3.6'
 services:
 
+  # !!! Containers in this file are meant to be started via `scripts/run-tests.ts`, not manully.
```

Comment:
Small typo "manully".

Reply:
Thanks, resolved in https://github.com/versionpress/versionpress/pull/1401/commits/8669fa4654aab8c19e9d00eee510ff98649f7dc9.